data_subscriptions/notifications/email_template.py

```diff
@@ -49,7 +49,10 @@ class DatasetActivity:
     def __init__(self, dataset, activities):
         self.dataset = dataset
         self.activities = activities
-        self.ckan_api = RemoteCKAN(CKAN_URL, apikey=CKAN_API_KEY)
+        self.ckan_api = RemoteCKAN(
+            CKAN_URL,
+            apikey=CKAN_API_KEY,
+            user_agent="data-subscription/latest (API call for CKAN activity pull)")
```

Comment:
@sagargg here it would be:

`data-subscription/latest (API call for single CKAN package activity)`